src/gpu/intel/ocl/gemm/gemm_with_post_ops.cl

```diff
@@ -98,7 +98,7 @@ __kernel void gemm_post_ops(__global SRC_DATA_T *src, __global BIA_DATA_T *bias,
 #else
     ACC_DATA_T acc = SRC_TO_ACC(src[data_idx]);
 #endif
-    float accumulator = acc;
+    float accumulator = convert_float(acc);
```

Comment:
unrelated to this issue but do we need to be concerned with potential f64 precision loss?

Reply:
💯 Yes.